back/requirements.txt

```diff
@@ -1,9 +1,92 @@
+appdirs==1.4.4
```

Comment:
Shouldn't exist so many dependences more than "django, restframework, and a couple more"

Reply:
Fixed !